CONTRIBUTING.md

```diff
@@ -62,18 +62,20 @@ To enable additional checks (including `-Werror` / `/WX` compilation flag), swit
 ["CMake standard options"](./README.md#cmake-standard-options) section in the top-level Readme.
 
 ### Code style
-
-We use `clang-format` to verify and apply code style changes. To see all rules we require,
-please take a look at `.clang-format` file in the root directory of this repository.
+We use `clang-format` to verify and apply code style changes to source files. 
+To see all rules we require, please take a look at `.clang-format` file in the 
+root directory of this repository. Similarry, we use `cmake-format` tool and
+`.cmake-format` file to verify and apply code style changes to CMake files.
 
 To enable code style checks and re-formatting, CMake option `UMF_FORMAT_CODE_STYLE` has to
-be switched on. You'll then have two additional CMake targets available.
+be switched on. You'll then have additional CMake targets available.
```

Comment:
actually, they are `make targets`, I guess

Reply:
I think that `CMake targets` is a good phrase to use here. These targets can be run ie. on Windows where CMake doesn't use `make` during build.

Reply:
Yeah, I know, but we use in a few places `make targest` and in other `CMake targets` - perhaps this is not that important